classes/requests/helpers/class-kp-order-data.php

```diff
@@ -198,7 +198,7 @@ public function get_klarna_order_line_object( $order_line ) {
 		}
 
 		$klarna_item = array(
-			'image_url'             => $order_line->get_image_url(),
+			'image_url'             => $this->maybe_allow_order_line_url( $order_line->get_image_url() ),
```

Comment:
@mntzrr An empty string will cause an error since the value wont be a valid URL according to Klarna and they will return a 400 bad value error response. Null is ok since the key will be stripped from the body [here](https://github.com/krokedil/klarna-payments-for-woocommerce/blob/develop-product-image-checkbox/classes/requests/helpers/class-kp-order-data.php#L229).

Reply:
My suggestion is to not add the `image_url` at all to the array if the function returns `null` as the field is optional.